src/NuGet.Core/NuGet.Packaging/LocalPackageArchiveDownloader.cs

```diff
@@ -144,23 +144,21 @@ public void Dispose()
 
             cancellationToken.ThrowIfCancellationRequested();
 
+            using (var source = File.OpenRead(_packageFilePath))
```

Comment:
It uses `useAsync` `false`.  Previously, the source was `_sourceStream.Value` which was created with `useAsync` `true`.